### DISCUSSION TITLE
How to use the function makeOffset

### DISCUSSION THREAD

**Discussion by raydeleu** [sgenoud/replicad#18] [How To]
The following code shows how to use the function `makeOffset`. Note that by creating an offset outwards, the shape is rounded on all edges with the same radius as the offset distance.  

<img width="542" alt="offset_replicad" src="https://user-images.githubusercontent.com/38007983/172070630-addeaadd-ce55-4648-8f2b-e4ec5de84299.png">

```` javascript
let box = new Sketcher("XY")
            .hLine(100)
            .vLine(60)
            .hLine(-100)
            .close()
            .extrude(30)
let boreHole = makeCylinder(40,40).translate([0,0,-5])
let boxGrown = makeOffset(box,3)
            .cut(box)
            .cut(boreHole)

````

**Comment by paulftw** [comment]
Thanks!
`makeOffset`'s [documentation](https://replicad.xyz/docs/api/#makeoffset)  says the first argument is a "Face", and it does work (sort of) when I pass it a Face, but this version is orders of magnitude more useful :)

Do you think the docs should be updated to mention the Shape as a potential first argument? 
cc @sgenoud

**Reply by sgenoud** [reply]
Good point!
